src/ble_security.rs

```diff
@@ -26,7 +26,16 @@ impl BLESecurity {
   }
 
   /// Set the passkey the server will ask for when pairing.
+  /// * The passkey returned should be between 000000..=999999 inclusive
+  /// and will always be exactly 6 digits, i.e. if 1234u32 is returned
+  /// the correct passkey will be '001234' and should be presented to the user padded to 6 digits
+  /// i.e. `format!("{:0>6}",pkey)`
```

Comment:
I think this can just say "The passkey will always be 6 digits. Setting the passkey to `1234` will actually make the passkey `"001234"`"